config/locales/de.helpers.yml

```diff
@@ -76,19 +76,31 @@ de:
     have_fun: "Viel Erfolg beim Training!"
     closing: "%{appname} Team"
   match_mailer:
-    next_game: "dein Spiel (%{team_home} vs. %{team_away}) im Event %{name}"
+    start_time: "Wann? Am %{start_date} um %{start_time} Uhr"
+    place: "Wo? %{place}"
+    here: "Hier"
+    apply_changes_html: "%{match_link} kannst du die Änderungen vornehmen."
+    add_results_html: "%{match_link} kannst du die Ergebnisse eintragen."
     match_notification:
-      info: "startet morgen um %{start_time}. Ihr trefft euch am folgenden Ort: %{place}"
-      subject: "Dein Spiel startet morgen"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} startet morgen."
```

Comment:
In my personal preference, due to the e-mail subjects being some sort of heading they should not have a full stop (.) at their end. But that's just my opinion. 😉

Reply:
@svitlanakalancha?